docs/user/exercises/programming-exercise-setup.rst

```diff
@@ -0,0 +1,175 @@
+Features
+^^^^^^^^
+
+- The support for a specific programming language depends on the used ``Continuous Integration`` system. The table below gives an overview:
+
+.. table:: ``Continuous Integration`` system support of programming languages
+
+  +----------------------+--------+---------+
+  | Programming Language | Bamboo | Jenkins |
+  +======================+========+=========+
+  | Java                 | true   | true    |
+  +----------------------+--------+---------+
+  | Python               | true   | true    |
+  +----------------------+--------+---------+
+  | C                    | true   | true    |
+  +----------------------+--------+---------+
+  | Haskell              | true   | true    |
+  +----------------------+--------+---------+
+  | Kotlin               | true   | false   |
+  +----------------------+--------+---------+
+  | VHDL                 | true   | false   |
+  +----------------------+--------+---------+
+  | Assembler            | true   | false   |
+  +----------------------+--------+---------+
+
+- Not all programming languages support the same feature set.
+  Depending on the feature set, some options might not be available during the creation of the programming exercise.
+  The table below provides an overview of the supported features:
+
+.. table:: Programming language feature sets
```

Comment:
This shows a GitHub Actions complaint: `Error parsing content block for the "table" directive: exactly one table expected.`

Reply:
I'm not sure why it doesn't accept this directive. The first one works just fine.

Reply:
Removed the directives as they added nothing. Should be fine now.